packages/iris-grid/src/mousehandlers/IrisGridTokenMouseHandler.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  type EventHandlerResult,
+  type Grid,
+  type GridPoint,
+  isLinkToken,
+  GridTokenMouseHandler,
+  type GridRangeIndex,
+} from '@deephaven/grid';
+import deepEqual from 'fast-deep-equal';
+import type IrisGrid from '../IrisGrid';
+
+// Handler also helps with other tooltips
+class IrisGridTokenMouseHandler extends GridTokenMouseHandler {
+  private irisGrid: IrisGrid;
+
+  private lastColumn: GridRangeIndex;
+
+  private lastRow: GridRangeIndex;
+
+  constructor(irisGrid: IrisGrid) {
+    super();
+
+    this.irisGrid = irisGrid;
+    this.lastColumn = null;
+    this.lastRow = null;
+  }
+
+  private destroyTooltip(): void {
+    this.irisGrid.setState({ hoverTooltipProps: null });
+  }
+
+  protected setCursor(gridPoint: GridPoint, grid: Grid): EventHandlerResult {
+    if (this.isHoveringLink(gridPoint, grid)) {
+      this.cursor = 'pointer';
+      return { stopPropagation: false, preventDefault: false };
+    }
+    this.cursor = null;
+    return false;
+  }
+
+  onMove(gridPoint: GridPoint, grid: Grid): EventHandlerResult {
+    const { model } = this.irisGrid.props;
+    const isUserHoveringLink = this.isHoveringLink(gridPoint, grid);
+    const tooltip = model.tooltipForCell(gridPoint.column, gridPoint.row);
+
+    if (
+      isUserHoveringLink &&
+      this.currentLinkBox != null &&
+      isLinkToken(this.currentLinkBox.token)
+    ) {
+      const { hoverTooltipProps } = this.irisGrid.state;
+      if (this.currentLinkBox == null) {
```

Comment:
Can this if statement ever be true?

Reply:
I think yes, in the following spots:

https://github.com/deephaven/web-client-ui/blob/36d910da75a0c628b9e45baf12282fb55e51e7e0/packages/grid/src/mouse-handlers/GridTokenMouseHandler.ts#L34
https://github.com/deephaven/web-client-ui/blob/36d910da75a0c628b9e45baf12282fb55e51e7e0/packages/grid/src/mouse-handlers/GridTokenMouseHandler.ts#L79

Reply:
Ignore, missed the earlier condition check